src/Redux/Actions/dashboard.action.js

```diff
@@ -0,0 +1,42 @@
+import jwtDecode from 'jwt-decode';
+import axios from 'axios';
+import baserUrl from '../../Api/config';
+
+const token = localStorage.getItem('barefoot_token');
+let loggedUser = '';
+if (token) loggedUser = jwtDecode(token);
+
```

Comment:
It would be better to use the existing helper function

Reply:
Great. done